fcc-test-suite-template-master/index.html

```diff
@@ -7,6 +7,7 @@
 	<title>Tribute to Steve Jobs</title>
```

Comment:
- [ ] Kindly try to use the Github inline comments in your review and make sure to add 2 comments at least